core/src/main/kotlin/org/evomaster/core/output/naming/ActionTestCaseNamingStrategy.kt

```diff
@@ -0,0 +1,27 @@
+package org.evomaster.core.output.naming
+
+import org.evomaster.core.problem.httpws.HttpWsCallResult
+import org.evomaster.core.problem.rest.RestCallAction
+import org.evomaster.core.search.EvaluatedIndividual
+import org.evomaster.core.search.Solution
+
+open class ActionTestCaseNamingStrategy(
+    solution: Solution<*>
+) : NumberedTestCaseNamingStrategy(solution)  {
+
+
+    override fun expandName(individual: EvaluatedIndividual<*>): String {
+        var evaluatedAction = individual.evaluatedMainActions().last()
+        var action = evaluatedAction.action as RestCallAction
+        var result = evaluatedAction.result as HttpWsCallResult
+        return "_${action.verb}_on_${sanitizePathName(action.path.nameQualifier)}_returns_${result.getStatusCode()}"
```

Comment:
@arcuri82 this first implementation only focuses in the verb, path and status code for REST individuals. Besides these three elements, which other aspects of a response do we wish to contemplate? 
Top of mind I'm thinking of an array response where we could add the `returns_empty`, `returns_2elements` instead of the status code. Not sure how easy it is to access that information from the individual, we hadn't discussed it in the original doc.

Reply:
an important aspect is to check if there is any fault, and, if so, put its code

Reply:
you can use `DetectedFaultUtils.getDetectedFaultCategories` to get such info

Reply:
Was actually thinking of adding that later to have more _to the point_ PRs, but sure. I'll add the field to represent the label, as we had originally discussed in the document.

Regarding these modifiers for the test name, do we wish to only name test cases using:
- status code for successful tests
- Fault information for tests containing faults
? Or do we wish to use extra metadata like in the first question? Maybe it can be something to evaluate later on

Reply:
How do we wish to handle a test case that has multiple faults? Currently there's no hierarchy, could the `FaultCategory.code` serve that purpose?

Reply:
@Pgarrett  yes, use unique sorted codes. see comments below